src/platform/user/authentication/utilities.js

```diff
@@ -237,22 +237,27 @@ export function sessionTypeUrl({
   // We should use OAuth when the following are true:
   // OAuth param is 'true'
   // config.OAuthEnabled is true
-  const useOAuth = useOauth || (config?.OAuthEnabled && OAuth === 'true');
+  const useOAuth =
+    useOauth || queryParams.oauth || (config?.OAuthEnabled && OAuth === 'true');
 
   // Only require verification when all of the following are true:
   // 1. On the USiP (Unified Sign In Page)
   // 2. The outbound application is one of the mobile apps
   // 3. The generated link type is for signup, and login only
+  // Explicitly determine OAuth usage
+
+  // Handle verification requirement
```

Comment:
We can revert this back as well since our `VerifyIdmeButton` and `VerifyLogingovButton` is handling it.